artemis-server/src/main/java/org/apache/activemq/artemis/core/paging/cursor/impl/PageCounterRebuildManager.java

```diff
@@ -216,6 +220,11 @@ public void rebuild() throws Exception {
          try (LinkedListIterator<PagedMessage> iter = msgs.iterator()) {
             while (iter.hasNext()) {
                PagedMessage msg = iter.next();
+               logger.info("Reading " + msg);
```

Comment:
Ditto

Reply:
It's debug.. not meant to be kept.. thank you